cursive/src/main.rs

```diff
@@ -788,29 +788,74 @@ fn get_translation_catalog() -> gettext::Catalog {
     gettext::Catalog::empty()
 }
 
-fn change_store(
-    mut ui: &mut Cursive,
-    valid_signing_keys: &Option<String>,
-    store_path: &str,
-    store: PasswordStoreType,
-) -> pass::Result<()> {
-    store
-        .lock()
-        .unwrap()
-        .reset(&Some(store_path.to_string()), valid_signing_keys)?;
+fn get_stores(
```

Comment:
This function seems like it belongs in the pass package, not in the cursive package

Reply:
I agree, but feel that we should rework the config handling even more. I would like to build the gui for updating the configuration before landing on how this will look.